### ISSUE TITLE
This no longer works !

### ISSUE THREAD

**Issue by ian10951** [zidansec/CrimeFlare#9]
All sites I've tried it states its unable to obtain the real IP address.

**Comment by zidansec** [comment]
Try using a VPN and running the tool again, or make sure you have php-curl installed.

**Comment by ian10951** [comment]
Yes I tried a VPN, several different servers actually, still the same result, and yes I have php-curl installed too.
